members/SNJG8473b29e.yaml

```diff
@@ -67,6 +67,8 @@ contact_form:
           selector: input[name='WFUD_Subject']
           value: $SUBJECT
           required: true
+          options:
+            max_length: 100
         - name: message
```

Comment:
is it just me or is the spacing off here? (grumble grumble stupid .yaml file format)

Reply:
Good call, though I think the spacing is OK. I think that `max_length` option has to be indented below `options`. I also double checked the other indentations in the yaml and saw that this part is not off. 

(just to triple-check Tom, could you please take a look at the yaml here, https://gist.github.com/OdeclasV/5cdbc09b33276cc76875fc5616336fc2 ?)

Reply:
When I eyeballed it it looked like the two `- name:` blocks for subject and message were not aligned, but when I put it into a text editor it looks fine, sorry!

Reply:
Np, thanks for checking!